keymanapp/keyman
# chore(oem/fv/ios): Remove build_keyboards.sh script

In preparation for #2036, this removes build_keyboards.sh script since the FV apps should now be standardized in using the utility `downloadKeyboardPackage()` for downloading fv_all.kmp.

This updates the iOS configure step to
```
cp "$KEYBOARDS_CSV" "$KEYBOARDS_CSV_TARGET"
```
which may temporarily diverge when keyboards.csv gets streamlined in number of columns.

## User Testing
* **TEST_FV_IOS** - Verifies FirstVoices iOS app
1. Install the PR build of FirstVoices on an iOS device/emulator
2. Launch FV and use the setup step steps to install a  keyboard: BC Coast --> SENCOTEN
3. From the setup step, enable FV as the default system keyboard
4. Launch a browser and with FV as a system keyboard, verify you can type with it.
